### PR TITLE
Python script to check Wangscape schemas and config

### PR DESCRIPTION
**Pull request by serin-delaunay**
This script does the following tasks:
* Verify that the Wangscape schemas conform to the draft 4 meta schema.
* Verify that a Wangscape options file conforms to the options schema.
* Verify that all module group files exist and confom to the module group schema.
* Verify that all source images exist.

There are some configuration problems that would not be detected by this script, such as:
* Source tile area not contained within the source image.
* Missing border/central module groups and no default module group.
* Missing input modules in the combiner module group print a message rather than raise an error (but they will cause Wangscape to crash).

However, I think this will detect the most common and egregious mistakes in configuration, and is enough to mark the fourth task in https://github.com/Wangscape/Wangscape/issues/21 as complete.

~Currently the options schema in master is broken;~ https://github.com/Wangscape/Wangscape/pull/112 fixes it.

I think it would also be worthwhile running this script on the example files as part of the Travis Wangscape builds.

### CODE REVIEW

**Review by hryniuk — APPROVED** [review]

**Comment by hryniuk** [comment]
I don't have write access to this repository, so I won't merge it.

**Comment by serin-delaunay** [comment]
Sorry about that. I've given you write access now. I'll merge this one.

**Pull request merged**
